src/ui/Tag.scss

```diff
@@ -0,0 +1,118 @@
+@import "variables";
```

Comment:
Why import variables.
Should you also use theme... to match with the Mendix theme? like  border-radius and colour
Many more relevant variables are available here: https://raw.githubusercontent.com/mendix/atlas-framework/master/_variables.scss